src/models/StyledComponent.js

```diff
@@ -13,16 +13,12 @@ import getComponentName from '../utils/getComponentName'
 import determineTheme from '../utils/determineTheme'
 import escape from '../utils/escape'
 import type { RuleSet, Target } from '../types'
-import { CONTEXT_KEY } from '../constants'
+import { CONTEXT_KEY, STATIC_EXECUTION_CONTEXT } from '../constants'
 
 import { CHANNEL, CHANNEL_NEXT, CONTEXT_CHANNEL_SHAPE } from './ThemeProvider'
 import StyleSheet from './StyleSheet'
 import ServerStyleSheet from './ServerStyleSheet'
 
-// HACK for generating all static styles without needing to allocate
```

Comment:
I see you moved this to constants.js, but you dropped the comment. I don't necessarily think it's a hack, but adding a comment that explains its use would be nice?

Reply:
Adressed via d9d6cd48e8b4a6611917f0e33406f70809b08f0b